src/pages/Thanks/ThanksPage.vue

```diff
@@ -540,6 +541,9 @@ export default {
 
 		this.loans = getLoans(this.receipt);
 
+		// Enable My Kiva Page for all users
+		this.myKivaFlagEnabled = readBoolSetting(data, MY_KIVA_FOR_ALL_USERS_KEY);
```

Comment:
Doesn't the portfolio/MyKiva page still need this change as well?

Reply:
That route doesn't exist, as we'll create a new one probably is fine for now...

Reply:
Ah that's right, we'll only need the experiment on the home page and here.